src/components/ProfileIcon/ProfileIcon.js

```diff
@@ -1,34 +1,17 @@
 import React from 'react'
 import { NavLink } from 'react-router-dom'
 import styled from 'styled-components/macro'
-import profilesicon from '../../img/profiles-icon.png'
+import profilesicon from '../../img/profiles-icon.svg'
 
 export default function ProfileIcon() {
   return (
-    <footer>
-      <LinkStyled
-        className="nav"
-        activeClassName="selected"
-        exact
-        to="/profiles"
-      >
-        <img src={profilesicon} alt="" />
-        <br />
-      </LinkStyled>
-      <span>Zu den Profilen</span>
-    </footer>
+    <LinkStyled activeClassName="selected" exact to="/profiles">
+      <img src={profilesicon} alt="" />
+      <br />
+    </LinkStyled>
   )
 }
 
 const LinkStyled = styled(NavLink)`
```

Comment:
kann man die Dateien anders organisieren, damit du nur in einem Dokument die NavLinks mit dem Style hast?

Reply:
Schaue ich nochmal, vorher war alles ganz anders, deswegen habe ich es in einzelnen Komponenten organisiert, jetzt könnte es wirklich auch anders gehen